view/about.twig

```diff
@@ -15,6 +15,7 @@
     <p>{% if ServiceName != 'SERVICE_NAME' %}{{ ServiceName }}{% else %}Skosmos{% endif %} {% trans "layout designed by Hahmo" %}</p>
   </div>
   <div class="version">
+    {% set version = request.version %}
```

Comment:
One can set a variable or use it directly, OK anyways. 😄

Reply:
Only simple variables can be used with `trans`. And the variable name has to match what has been used before, otherwise all the translated strings would be invalidated.

Yes, a global variable would have been possible too. I decided to tack this into Request, since it's passed to all templates anyway and it already had a reference to Model, so it was trivial to delegate to Model->getVersion.